lib/cmr/cmr_client.rb

```diff
@@ -736,6 +736,49 @@ def create_update_order_option(order_option:, provider_id:, native_id:, token:)
       post(url, order_option.to_json, headers.merge(token_header(token)))
     end
 
+    def get_data_quality_summaries(provider_id:, concept_id: nil, token:)
+      options = {}
+      options[:provider_id] = provider_id
+      options[:concept_id] = concept_id if concept_id
+      options[:page_size] = 2000
+      get('/search/data-quality-summaries.umm_json', options, token_header(token))
+    end
+
+    def create_update_data_quality_summary(data_quality_summary:, provider_id:, native_id:, token:)
+      url = "/ingest/providers/#{provider_id}/data-quality-summaries/#{encode_if_needed(native_id)}"
+      headers = {
+        'Accept' => 'application/json; charset=utf-8',
+        'Content-Type' => 'application/json'
+      }
+      post(url, data_quality_summary.to_json, headers.merge(token_header(token)))
+    end
+
+    def remove_data_quality_summary(provider_id:, native_id:, token:)
+      url = "/ingest/providers/#{provider_id}/data-quality-summaries/#{encode_if_needed(native_id)}"
+      headers = { 'Accept' => 'application/json; charset=utf-8' }
+      delete(url, {}, nil, headers.merge(token_header(token)))
+    end
+
+    def create_collection_association(collection_concept_id:, concept_id:, token:)
+      url = "/search/associate/#{provider_id}/#{collection_concept_id}"
+      payload = []
+      concept = {}
+      concept[:concept_id] = concept_id
+      payload << concept
+      headers = { 'Accept' => 'application/json; charset=utf-8' }
+      post(url, payload.to_json, headers.merge(token_header(token)))
+    end
+
+    def remove_collection_association(collection_concept_id:, concept_id:, token:)
+      url = "/search/associate/#{provider_id}/#{collection_concept_id}"
+      payload = []
+      concept = {}
+      concept[:concept_id] = concept_id
```

Comment:
This concept_id is the data-quality-sumamry concept_id, correct?

Reply:
yes, that is dqs concept_id. Let me remove this and the other method (create_collection_association and remove_collection_association) from the PR, because they don't belong to this ticket.

Reply:
Removed